.test-infra/jenkins/LoadTestsBuilder.groovy

```diff
@@ -43,7 +43,7 @@ class LoadTestsBuilder {
 
 
   static void loadTest(context, String title, Runner runner, SDK sdk, Map<String, ?> options,
-      String mainClass, List<String> jobSpecificSwitches = null) {
+      String mainClass, List<String> jobSpecificSwitches = null, String requirementsFile = null) {
```

Comment:
is the requirements file the same as a requirements.txt file?  Optional- call it requirementsTxtFile then and I think it should be clear to beam devs.